infra/token-transfer-server/src/enums.js

```diff
@@ -0,0 +1,31 @@
+/**
+ * IMPORTANT: If you add an entry to an enum below, do not forget to add
+ *  a migration script to add the enum to the DB.
+ */
+
+class Enum extends Array {
+  constructor(...args) {
+    super(...args)
+
+    for (const k of args) {
+      this[k] = k
+    }
+  }
+}
+
+const TransferStatuses = new Enum(
+  // Transfer requested by user.
+  'Enqueued',
+  // Transfer transaction sent to the network. Waiting for confirmation.
+  'WaitingConfirmation',
+  // Transfer successfully executed by the network.
+  'Success',
+  // Transfer transaction failed.
+  'Failed',
+  // Transfer cancelled by the user.
+  'Cancelled'
```

Comment:
Do you think we need an additional one called `Paused` or something similar, for if the transfer was paused by us if we want to do some manual verification?

Reply:
Yes, good idea - it could be handy... Done.